### PR TITLE
Changes for release v0.17.0

### PR DESCRIPTION
**Pull request by daniel-de-vera**
## Change description

Changes for release v0.17.0

### CODE REVIEW

File: signadot/operator/templates/signadot-manager-namespaced-clusterrole.yaml [hunk 1]
```diff
@@ -47,6 +50,7 @@ rules:
   - apps
   resources:
   - deployments
+  - statefulsets
```
Comment (scott-cotton) [inline signadot/operator/templates/signadot-manager-namespaced-clusterrole.yaml:53]: I don't think we need this anymore.

**Review by scott-cotton — APPROVED** [review]
LGTM

not sure what to do about the statefulsets.  I'm ok with removing them next time. wdyt?

**Comment by daniel-de-vera** [comment]
> not sure what to do about the statefulsets. I'm ok with removing them next time. wdyt?

Yes, or I can manually remove it from the chart, and then send a PR for updating `signadot` repo. I thought those rbacs were auto-generated.

**Comment by scott-cotton** [comment]
> > not sure what to do about the statefulsets. I'm ok with removing them next time. wdyt?
> 
> Yes, or I can manually remove it from the chart, and then send a PR for updating `signadot` repo. I thought those rbacs were auto-generated.

I'm fine with whatever you choose.

**Pull request merged**